### PR TITLE
Make rubocop green again

### PR DESCRIPTION
**Pull request by Flipez**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,29 +1,31 @@
+# frozen_string_literal: true
+
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-gem 'rails', '~> 5.2.3'
-gem 'sqlite3'
+gem 'execjs'
```
Comment (bastelfreak) [inline Gemfile]: Is there a reason for all the reordering? I never noticed a relatec rubocop rule for this.

File: app/controllers/dashboard_controller.rb [hunk 2]
```diff
@@ -1,10 +1,12 @@
+# frozen_string_literal: true
+
 class DashboardController < ApplicationController
   def show
     @data = begin
       JSON.parse(RedisClient.client.get('repo_status_data').to_s)
-    rescue JSON::ParserError
-      nil
+            rescue JSON::ParserError
```
Comment (bastelfreak) [inline app/controllers/dashboard_controller.rb]: this is probably correct, but it looks wrong :(

File: bin/webpack-dev-server [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
-ENV["NODE_ENV"]  ||= "development"
+ENV['RAILS_ENV'] ||= ENV['RACK_ENV'] || 'development'
+ENV['NODE_ENV']  ||= 'development'
 
-require "pathname"
-ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../Gemfile",
-  Pathname.new(__FILE__).realpath)
+require 'pathname'
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile',
+                                           Pathname.new(__FILE__).realpath)
```
Comment (bastelfreak) [inline bin/webpack-dev-server]: can't we just put it into a single line?

File: config/environments/production.rb [hunk 4]
```diff
@@ -1,94 +1,38 @@
...
-
-  # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  # config.action_controller.asset_host = 'http://assets.example.com'
-
-  # Specifies the header that your server uses for sending files.
-  # config.action_dispatch.x_sendfile_header = 'X-Sendfile' # for Apache
-  # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
-
-  # Store uploaded files on the local file system (see config/storage.yml for options)
   config.active_storage.service = :local
```
Comment (bastelfreak) [inline config/environments/production.rb]: is it required to drop all the commets? I think they are quite useful

File: Gemfile (hunk 1 above)
Comment (Flipez) [reply Gemfile]: Yes, there is a rubocop rule enforcing this.

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Bundler/OrderedGems

File: config/environments/production.rb (hunk 4 above)
Comment (Flipez) [reply config/environments/production.rb]: Just a matter of taste. I've restored them

**Comment by Flipez** [comment]
Not all cases fixed, but I think the majority is enough for now.

**Pull request merged**